model/src/gateway/event/mod.rs

```diff
@@ -349,3 +349,73 @@ impl Display for EventConversionError {
 }
 
 impl Error for EventConversionError {}
+
+#[cfg(test)]
+mod tests {
+    use super::{super::payload::incoming::*, shard::*, Event};
```

Comment:
Can we add `tests` module docs on what to do if one of these tests fails?

Reply:
It's somewhat complex and I don't know exactly how to phrase it. I.e. when should `EVENT_THRESHOLD` be changed due to an event being very close to it in size

Reply:
I believe what they mean is to add a note saying that if a model changes to the point that this fails then it should be `Box`ed